.github/workflows/build_and_test.yml

```diff
@@ -39,10 +39,10 @@ jobs:
     needs: params
     runs-on: ubuntu-20.04
     container:
-      image: ${{ needs.params.outputs.build_image_name }}:latest
```

Comment:
Why is this needed? It seem unrelated. If it is needed, then that's something that would be helpful to explain in the PR description.